site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -48,6 +52,104 @@ private function validateDiffViewerType(string $type) {
         return true;
     }
 
+    
+    /**
+     * Route for randomizing peer assignments with 'One Grades Many'
+     *
+     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/ajaxRandomizePeers", methods={"POST"})
```

Comment:
We don't usually have the word `ajax` in our routes. Could you modify this to just `RandomizePeers`?

Reply:
Done!